kobo/apps/project_ownership/docs/api/v2/project_ownership/invites/update.md

```diff
@@ -0,0 +1,13 @@
+## Update an invite status
+
+Update the status of an invite.
+Status accepted:
+- `cancelled`
+- `accepted`
+- `declined`
+
+**Notes**:
+- _When submitting `accepted` the invite status becomes automatically `in_progress`_
+- _Only the sender can cancel an invite, and **if only if** the invite is still pending._
+- _Only the recipient can accept or decline, **if and only if** the invite is still pending._
+
```

Comment:
Also same error issue as with the POST and DELETE requests. I can't make a PATCH without getting this error but it works fine locally. 

<img width="1094" height="711" alt="Image" src="https://github.com/user-attachments/assets/674374de-a511-46db-902b-24151da14fde" />